fhir-persistence-jdbc/src/test/java/com/ibm/fhir/persistence/jdbc/test/spec/R4JDBCExamplesTest.java

```diff
@@ -92,7 +115,8 @@ protected FHIRPersistenceContext createHistoryPersistenceContext() {
 
     @Override
     public FHIRPersistence getPersistenceImpl() throws Exception {
-        return new FHIRPersistenceJDBCImpl(this.properties, database);
+        //return new FHIRPersistenceJDBCImpl(this.properties, database);
```

Comment:
what's this commented for?

Reply:
good catch, I mean to remove it. will remove

Reply:
done